packages/emoji-mart/src/components/Emoji/EmojiProps.js

```diff
@@ -17,6 +17,10 @@ export default {
     },
   },
 
+  imageURL: null,
+  spritesheetURL: null,
```

Comment:
`imageURL` and `spritesheetURL` would need to be added to the README.

Reply:
@EtienneLem It seems getProps will only allow defaultProps regardless of what props we pass. I am passing `spritesheet:true` in order to use spritesheet image as source. But `getProps()` function is only generating props using defaultProps where spritesheet prop is not present!

what am I missing here!?

Could you please help? @EtienneLem